### ISSUE TITLE
`comma-dangle`:  TSTypeParameterDeclaration in tsx

### ISSUE THREAD

**Issue by hyoban** [eslint-stylistic/eslint-stylistic#1126]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

Without the comma, this is an invalid syntax in TSX.

### Reproduction

https://typescript-eslint.io/play#ts=5.8.2&showAST=es&fileType=.tsx&code=FAYw9gdgzgLgBAQzgXjgHgCoD4AUBKFLOfYIA&eslintrc=N4KABGBEBOCuA2BTAzpAXGYBfEWg&tsconfig=N4KABGBEDGD2C2AHAlgGwKYCcDyiAuysAdgM6QBcYoEEkJemy0eAcgK6qoDCAFutAGsylBm3TgwAXxCSgA&tokens=false

### System Info

```shell
System:
    OS: macOS 26.2
    CPU: (12) arm64 Apple M4 Pro
    Memory: 448.39 MB / 24.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 24.13.0 - /Users/hyoban/n/bin/node
    npm: 11.6.2 - /Users/hyoban/n/bin/npm
    pnpm: 10.27.0 - /Users/hyoban/n/bin/pnpm
    bun: 1.3.5 - /Users/hyoban/.bun/bin/bun
  npmPackages:
    @typescript-eslint/parser: 8.55.0 => 8.55.0 
    eslint: 10.0.0 => 10.0.0
```

### Contributes

- [x] I am willing to submit a PR to fix this issue
- [x] I am willing to submit a PR with failing tests

**Comment by 9romise** [comment]
Thank you for reporting this issue. I’ve found and fixed it in a branch already.
See https://github.com/eslint-stylistic/eslint-stylistic/pull/1066#discussion_r2780786734
If you’d like, feel free to submit a PR for it — otherwise, I’ll submit one separately to address the fix shortly.
